cmd/run/run.go

```diff
@@ -337,13 +352,16 @@ type Config struct {
 // DefaultConfig returns the OpenFGA server default configurations.
 func DefaultConfig() *Config {
 	return &Config{
-		MaxTuplesPerWrite:             100,
-		MaxTypesPerAuthorizationModel: 100,
-		ChangelogHorizonOffset:        0,
-		ResolveNodeLimit:              25,
-		Experimentals:                 []string{},
-		ListObjectsDeadline:           3 * time.Second, // there is a 3-second timeout elsewhere
-		ListObjectsMaxResults:         1000,
+		MaxTuplesPerWrite:                100,
+		MaxTypesPerAuthorizationModel:    100,
+		MaxConcurrentReadsForCheck:       30, // same as Datastore.MaxOpenConns
```

Comment:
Should we have this value reflect the `--datastore-max-open-conns` if this config is not provided?

e.g. `openfga run --datastore-max-open-conns=100` then that implies `--max-concurrent-reads-for-check=100`

That way we're not hindering/impacting latency with a more restrictive read limit for developers using OpenFGA if they don't have database issues.

Reply:
@jon-whit it would be safer to limit them, as you do not 1 intensive check to be blocking all other checks happening at the same time. People who do not have this issue can increase it, but I think this is better as a default.